### PR TITLE
bump esmini version

### PR DESCRIPTION
**Pull request by Robert108**

### CODE REVIEW

**Comment by Robert108** [comment]
Previous version is no longer hosted on esmini github

File: atos/modules/EsminiAdapter/inc/esminiadapter.hpp [hunk 1]
```diff
@@ -81,7 +81,7 @@ class EsminiAdapter : public Module {
 	static std::pair<uint32_t, std::string> parseAction(const std::string& action);
 	static bool isStartAction(const std::string& action);
 	static bool isSendDenmAction(const std::string& action);
-	static void collectStartAction(const char* name, int type, int state);
+	static void collectStartAction(const char* name, int type, int state, const char* full_path);
```
Comment (victorjarlow) [inline atos/modules/EsminiAdapter/inc/esminiadapter.hpp:84]: Was this added in the new version of esmini?

**Review by victorjarlow — APPROVED** [review]

**Review by SebastianSvanland — APPROVED** [review]

File: atos/modules/EsminiAdapter/src/esminiadapter.cpp [hunk 2]
```diff
@@ -281,26 +283,28 @@ void EsminiAdapter::collectStartAction(
...
+	int state,
+	const char *full_path)
 {
-	RCLCPP_DEBUG(me->get_logger(), "Storyboard state changed! Name: %s, Type: %d, State: %d", name, type, state);
+	RCLCPP_DEBUG(me->get_logger(), "Storyboard state changed! Name: %s, Type: %d, State: %d, Full path: %s", name, type, state, full_path);
 	// switch on type
 	switch (type)
 	{
-	case 6: // Action
+	case 7: // Action
```
Comment (samuelthoren) [inline atos/modules/EsminiAdapter/src/esminiadapter.cpp:299]: Is there a reason the cases aren't in order?

**Review by samuelthoren — APPROVED** [review]

File: atos/modules/EsminiAdapter/src/esminiadapter.cpp [hunk 3]
```diff
@@ -256,15 +256,17 @@ bool EsminiAdapter::isSendDenmAction(const std::string& action)
 /*!
  * \brief Add delayed start to object state if start action occurred.
  * \param name Name of the StoryBoardElement whose state has changed.
- * \param type Possible values: STORY = 1, ACT = 2, MANEUVER_GROUP = 3, MANEUVER = 4, EVENT = 5, ACTION = 6, UNDEFINED_ELEMENT_TYPE = 0.
+ * \param type Possible values: STORY_BOARD = 1, STORY=2, ACT = 3, MANEUVER_GROUP = 4, MANEUVER = 5, EVENT = 6, ACTION = 7, UNDEFINED_ELEMENT_TYPE = 0.
```
Comment (victorjarlow) [inline atos/modules/EsminiAdapter/src/esminiadapter.cpp]:
```suggestion
 * \param type Possible values: STORY_BOARD = 1, STORY = 2, ACT = 3, MANEUVER_GROUP = 4, MANEUVER = 5, EVENT = 6, ACTION = 7, UNDEFINED_ELEMENT_TYPE = 0.
```

**Review by victorjarlow — APPROVED** [review]

**Pull request merged**